torch/_dynamo/symbolic_convert.py

```diff
@@ -2084,6 +2086,16 @@ def init_local_index_guards_hack(self):
                 ]
                 self.output.guards.update(index_guards)
 
+    def collect_debug_info_from_symbolic_locals(self, f_locals):
+        # Count number of SymNodeVariable in symbolic_locals where the value is a list/tuple
```

Comment:
This seems... insufficient? Like in the KJT case, the list of ints will be nested inside of a UserDefinedObjectVariable